packages/vm/src/evm/opcodes/util.ts

```diff
@@ -358,12 +358,12 @@ export const eof1ValidOpcodes = (code: Buffer) => {
       // Skip data block following push
       x += opcode - 0x5f
       if (x > code.length - 1) {
-        // Push blocks mmust not exceed end of code section
+        // Push blocks must not exceed end of code section
         return false
       }
     }
   }
-  const terminatingOpcodes = new Set([0x00, 0xd3, 0xfd, 0xfe, 0xff])
+  const terminatingOpcodes = new Set([0x00, 0xf3, 0xfd, 0xfe, 0xff])
```

Comment:
Oh wow, missed this 👍

Reply:
Yeah, me too. But writing those new tests you suggested helped me catch it 🎆